apps/meteor/client/hooks/useFeaturePreviewEnableQuery.ts

```diff
@@ -0,0 +1,27 @@
+import type { FeaturePreviewProps } from '@rocket.chat/ui-client';
+
+export const useFeaturePreviewEnableQuery = (features: FeaturePreviewProps[]) => {
```

Comment:
if you are creating a hook it should have hooks inside, otherwise is only a regular function